src/cascadia/CascadiaPackage/Package-Can.appxmanifest

```diff
@@ -111,8 +111,8 @@
             <com:ComInterface>
                 <com:ProxyStub Id="1D1852F4-ADAD-42B6-9A43-9437AAAD7717" DisplayName="OpenConsoleHandoffProxy" Path="OpenConsoleProxy.dll"/>
                 <com:Interface Id="E686C757-9A35-4A1C-B3CE-0BCC8B5C69F4" ProxyStubClsid="1D1852F4-ADAD-42B6-9A43-9437AAAD7717"/>
-                <com:Interface Id="59D55CCE-FC8A-48B4-ACE8-0A9286C6557F" ProxyStubClsid="1D1852F4-ADAD-42B6-9A43-9437AAAD7717"/> <!-- ITerminalHandoff -->

```

Comment:
This interface is essentially an internal currency between OpenConsole and the terminal. It doesn't need to support all interfaces. And after this PR OpenConsole will only support `ITerminalHandoff3`.

But Dustin "Human Wikipedia" Howett explained me how stubs work and basically per user there can only be 1 single COM proxy stub instance for Windows Terminal. So, if the Windows Terminal Canary stub gets installed and it only supported `ITerminalHandoff3` then any older version of Windows Terminal that only supports `ITerminalHandoff2` would not work anymore.

However, there's some mapping from interface GUIs to proxies so it may just work to remove them anyway. He said we should just try it.